pkg/querier/queryrange/cache_extractor.go

```diff
@@ -0,0 +1,19 @@
+package queryrange
+
+import (
+	"github.com/cortexproject/cortex/pkg/querier/queryrange"
+)
+
+// SeriesExtractor implements Extractor interface
+type SeriesExtractor struct{}
```

Comment:
nit: I'd probably rename this `DefaultExtractor` as it's reusable and doesn't have any logic tied to series.